src/tools/miropt-test-tools/src/lib.rs

```diff
@@ -4,6 +4,8 @@ pub struct MiroptTestFiles {
     pub expected_file: std::path::PathBuf,
     pub from_file: String,
     pub to_file: Option<String>,
+    pub passes: Vec<String>,
+    //Vec of passes under test to be dumped
```

Comment:
We usually put documentation comments before the thing being documented. You can even use `///` in that case to make it show up in internal documentation